projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -505,6 +505,11 @@ ip_nat_pool
 :
    POOL name = variable PREFIX_LENGTH prefix_length = DEC NEWLINE
    ip_nat_pool_range*
+   | POOL name = variable first = IP_ADDRESS last = IP_ADDRESS
```

Comment:
If your tests pass I would imagine this is technically correct. However it is very hard to discern what is going on here. Can you please add parentheses around each alternative of the pipe for clarity?